contracts/src/CrossChainApplications/examples/NativeTokenBridge/NativeTokenDestination.sol

```diff
@@ -14,6 +14,9 @@ import {INativeTokenDestination} from "./INativeTokenDestination.sol";
 import {ITokenSource} from "./ITokenSource.sol";
 import {TeleporterFeeInfo, TeleporterMessageInput} from "@teleporter/ITeleporterMessenger.sol";
 import {TeleporterOwnerUpgradeable} from "@teleporter/upgrades/TeleporterOwnerUpgradeable.sol";
+// TeleporterUpgradeable import is used for natspec importdoc
+// solhint-disable-next-line no-unused-import
+import {TeleporterUpgradeable} from "@teleporter/upgrades/TeleporterUpgradeable.sol";
```

Comment:
have you tested with the generated `solc` docs whether it's necessary to use `@inheritdoc`? Seems it might do so by default 
> Functions without NatSpec will automatically inherit the documentation of their base function. 

https://docs.soliditylang.org/en/latest/natspec-format.html